Document/0x05f-Testing-Local-Authentication.md

```diff
@@ -4,6 +4,100 @@ During local authentication, an app authenticates the user against credentials s
 As described earlier in Testing Authentication and Session Management: it is important to reassure that authentication happens at least on a cryptographic primitve (e.g.: an authentication step which results in unlocking a key). Next, it is recommended that the authentication is verified at a remote endpoint.
 In Android, there are two mechanisms supported by the Android Runtime for local authentication: the Confirm Credential flow and the Biometric Authentication flow.
 
+### Testing App Permissions

```

Comment:
This section does not really fit in local authentication, as permissions do not authenticate the user, but rather in H: testing app permissions. Over there, we already have a section for Testing App Permissions. However: that section needs to be updated given the changes provided by api level 24-28... So, can you instead please update that section?

Next platform overview (0x5a) needs a small update given the changes of how permissions are used: over there we do talk about certain permissions which are no longer available in certain api-versions that will require updating as well.

Reply:
I understand what needs to be updated now. Thank you.